.github/workflows/sca.yml

```diff
@@ -29,6 +29,13 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      # We need to fetch base branch because `actions/checkout` only initializes empty repo and fetches PR's meta-branch
+      # which leads to `fatal: ambiguous argument 'origin/...': unknown revision or path not in the working tree.`,
```

Comment:
👍🏻 

I wonder why this fatal was not breaking this workflow in first place

Reply:
I don't know, did not research it this direction, I just wanted to fix Mess Detector. It should fail, though, I agree.